tests/unit/core/utils/test_index.js

```diff
@@ -839,3 +838,20 @@ describe('sanitizeUserHTML', () => {
     expect(sanitize(null)).toEqual('');
   });
 });
+
+describe('sanitizeHTML', () => {
+  it('does not change links', () => {
+    const html = '<a href="http://example.org">link</a>';
+    expect(sanitizeHTML(html, ['a'])).toEqual({
+      __html: html,
+    });
+  });
+
+  // This is a built-in feature of dompurify.
+  it('removes `target` attribute on links', () => {
```

Comment:
Could this test fail if the disco test ran before it and modified the purify instance adding the hook?

Reply:
ah oops, I guess yes.